helm-service/main.go

```diff
@@ -191,6 +191,12 @@ func _main(args []string, env envConfig) int {
 	return 0
 }
 
+func healthEndpointHandler(w http.ResponseWriter, r *http.Request) {
+	if r.URL.Path == "/health" {
+		w.WriteHeader(http.StatusOK)
+	}
+}
+
```

Comment:
These lines are used in multiple places. Would it make sense to move it in a common package, such as `go-utils`?

Reply:
back then I was not 100% sure if I will end up differentiating between health/ready and health/lives but since now they are the same, sure I can add it to that one